code/modules/research/xenoarchaeology/artifact/effects/unknown_effect_menagerie.dm

```diff
@@ -6,7 +6,7 @@
 
 /datum/artifact_effect/menagerie/New()
 	..()
-	possible_types = existing_typesof(/mob/living) - existing_typesof(/mob/living/silicon)
+	possible_types = existing_typesof(/mob/living) - (existing_typesof(/mob/living/silicon) + existing_typesof(/mob/living/simple_animal/hostile/humanoid))
```

Comment:
I'm not certain, but I'm pretty sure when subtracting multiple things from a list you want to do `list - A - B - C`.